themes/_core/js/theme.js

```diff
@@ -4,7 +4,6 @@ window.$ = $;
 window.jQuery = $;
 
 import './migrate-mute'
-import 'jquery-migrate';
```

Comment:
Then when is `jquery-migrate` included I found an other reference in classic theme
If so maybe it's better to remove it in the theme, this way the core dependency will still include it even if one use another theme

Reply:
yep you're right actually, just fixed it 👍